### PR TITLE
Branch event

### LINKED ISSUE #18: As a user, I can view events sorted by priority.
As a user, I can view events sorted by priority, so that I can be aware of important events.

### LINKED ISSUE #28: As a user, I can add new events.
As a user, I can add new events, so that I can keep track of them.

### PR DESCRIPTION
**Pull request by GanapathySanathBalaji**
Add functionality related to events.

### CODE REVIEW

File: src/main/java/event/EventList.java [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    }
+
+    /**
+     * Adds a new event to the list.
+     *
+     * @param newEvent Event represents the new event tot be added.
+     */
+    public void addEvent(Event newEvent) {
+        ui.printLine();
+        System.out.println("A new event with the following information has been added.");
```
Comment (terrytay) [inline src/main/java/event/EventList.java:35]: Please put this message after the event has been successful added

File: src/main/java/event/EventList.java [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+        System.out.println(newEvent.getEventInformation());
+        ui.printLine();
+        events.add(newEvent);
```
Comment (terrytay) [inline src/main/java/event/EventList.java]: line 33 should come after this

File: src/main/java/event/EventList.java (hunk 1 above)
Comment (GanapathySanathBalaji) [reply src/main/java/event/EventList.java:35]: Ok, will update as required.

Comment (terrytay) [reply src/main/java/event/EventList.java:35]: is ok ive done it

**Review by terrytay — APPROVED** [review]
Thanks for the good work

**Pull request merged**